unit-tests/live/streaming/test-y16-calibration-format.py

```diff
@@ -0,0 +1,57 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+# This test check streaming y16 profile
+
+import time
+import pyrealsense2 as rs
+from rspy import test, log
+from rspy.timer import Timer
+
+
+y16_streamed = False
+
+
+def close_resources(sensor):
+    """
+    Stop and Close sensor.
+    :sensor: sensor of device
+    """
+    if len(sensor.get_active_streams()) > 0:
+        sensor.stop()
+        sensor.close()
+
+
+def frame_callback(frame):
+    global y16_streamed
+    y16_streamed = True
+
+
+timer = Timer(30)
+
+device = test.find_first_device_or_exit()
+depth_sensor = device.first_depth_sensor()
+
+test.start('Check that y16 is streaming:')
+
+profile_y16 = next(p for p in depth_sensor.profiles if p.format() == rs.format.y16)
+test.check(profile_y16)
```

Comment:
This checking should fail since no streaming has started yet.
Please remove it.